packages/framework/tree-agent/.mocharc.cjs

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+"use strict";
+
+const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
+
+const packageDir = __dirname;
+const config = getFluidTestMochaConfig(packageDir);
+config.spec = process.env.MOCHA_SPEC ?? "lib/test";
+// TODO: figure out why this package needs the --exit flag, tests might not be cleaning up correctly after themselves
+// AB#7856
```

Comment:
Relevant to this package? Or copied from another package?

Reply:
I removed it for now.